src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/ListViewTests.cs

```diff
@@ -0,0 +1,511 @@
+                Assert.Equal(3, accessibleObject.GetChildIndex(listView.Items[0].SubItems[3].AccessibilityObject));
+                Assert.Equal(4, accessibleObject.GetChildIndex(listView.Items[0].SubItems[4].AccessibilityObject));
+                Assert.Equal(5, accessibleObject.GetChildIndex(listView.Items[0].SubItems[5].AccessibilityObject));
+                Assert.Equal(-1, accessibleObject.GetChildIndex(listView.Items[0].SubItems[6].AccessibilityObject));
+                Assert.Equal(-1, accessibleObject.GetChildIndex(listView.Items[0].SubItems[7].AccessibilityObject));
+                Assert.Equal(-1, accessibleObject.GetChildIndex(listView.Items[0].SubItems[8].AccessibilityObject));
+                Assert.Equal(-1, accessibleObject.GetChildIndex(listView.Items[0].SubItems[9].AccessibilityObject));
+                Assert.Equal(-1, accessibleObject.GetChildIndex(listView.Items[0].SubItems[10].AccessibilityObject));
+
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task ListView_Tile_ColumnProperty_ReturnsMinusOneAsync()
+        {
+            await RunTestAsync((form, listView) =>
+            {
+                InitializeTileList(listView, columnCount: 4, subItemsCount: 3, tileSize: new Size(150, 150));
+                Application.DoEvents();
+                Assert.Equal(-1, listView.Items[0].SubItems[0].AccessibilityObject.Column);
+                Assert.Equal(-1, listView.Items[0].SubItems[1].AccessibilityObject.Column);
+                Assert.Equal(-1, listView.Items[0].SubItems[2].AccessibilityObject.Column);
+                Assert.Equal(-1, listView.Items[0].SubItems[2].AccessibilityObject.Column);
+
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task Click_On_Second_Column_Does_Not_Alter_CheckBoxesAsync()
+        {
+            await RunTestAsync(async (form, listView) =>
+            {
+                InitializeItems(listView, View.Details, virtualModeEnabled: false, checkBoxesEnabled: true);
+                foreach (ListViewItem item in listView.Items)
+                {
+                    Assert.Equal(0, item.StateImageIndex);
+                }
+
+                foreach (ListViewItem item in listView.Items)
+                {
+                    Assert.False(item.Selected);
+                }
+
+                Point listViewCenter = GetCenter(listView.RectangleToScreen(listView.Items[0].SubItems[1].Bounds));
+
+                await MoveMouseAsync(form, listViewCenter);
+
+                await InputSimulator.SendAsync(
+                   form,
+                   inputSimulator => inputSimulator.Keyboard.KeyDown(VirtualKeyCode.SHIFT));
+
+                await InputSimulator.SendAsync(
+                    form,
+                    inputSimulator => inputSimulator.Mouse.LeftButtonClick());
```

Comment:
I suspect these can be amalgamated together:
```suggestion
                await InputSimulator.SendAsync(
                   form,
                   inputSimulator => 
                       inputSimulator.Keyboard.KeyDown(VirtualKeyCode.SHIFT))
                                     .Mouse.LeftButtonClick());
```